solutionbox/inception/datalab_solutions/inception/_cloud.py

```diff
@@ -103,10 +102,9 @@ def train(self, input_dir, batch_size, max_steps, output_path,
     job_request = {
       'package_uris': staging_package_url,
       'python_module': 'datalab_solutions.inception.task',
-      'scale_tier': scale_tier,
-      'region': region,
       'args': job_args
     }
+    job_request.update(dict(cloud_train_config._asdict()))
```

Comment:
does the service print a clear error message if both scale tier and worker_count are set?